tox.ini

```diff
@@ -16,10 +16,9 @@ commands =
 
 [testenv:cloud_tests]
 passenv =
-    MARQO_CLOUD_URL
-    MARQO_API_KEY
-    MARQO_URL
-    TEST_INDEX_SUFFIX
+    MARQO_CLOUD_URL # URL that config uses to resolve whether cluster is cloud v2
+    MARQO_API_KEY # This is the API key used to authenticate with the cloud
+    MARQO_URL # URL that is used as marqo instance url
 whitelist_externals =
   python
 deps =
```

Comment:
do we need to add `zlib` somewhere? As it used hashing index settings?

Reply:
I believe that zlib is a built-in library